docs/vectorizer/worker.md

```diff
@@ -1,53 +1,136 @@
-loops over the vectorizers defined in your database and processes each vectorizer in turn.
 
-This page shows you how to install, run, and manage the workers that run vectorizers in your database:
+# Running on self-hosted Postgres or other platforms
 
-- [Install and configure vectorizer worker](#install-and-configure-vectorizer-worker): setup the environment
-  to securely run vectorizers defined in a self-hosted Postgres environment
-- [Run vectorizers with vectorizer worker](#run-vectorizers-with-vectorizer-worker): run specific
-  vectorizers in your database as either single, parallel or concurrent tasks
-- [Set the time between vectorizer worker runs](#set-the-time-between-vectorizer-worker-runs): manage run scheduling
-- [Additional configuration via environment variables](#additional-configuration-via-environment-variables) an overview of the environment variables and their purpose
+When you use pgai vectorizers on a self-hosted Postgres installation or another cloud provider other than Timescale Cloud, you have to run the vectorizer worker yourself. The vectorizer worker will connect to your database and process the vectorizers you have defined. You can run the vectorizer:
+- Through the pgai CLI tool as `pgai vectorizer worker` (see [instructions below](#running-a-vectorizer-worker-as-a-cli-tool))
+- Integrating the vectorizer worker as a background process into your own python application (see [instructions below](#running-a-vectorizer-worker-in-your-own-application))
+- Using the vectorizer worker Docker image (see [instructions below](#running-a-vectorizer-worker-with-docker))
+- In a Docker Compose configuration (see [instructions below](#running-a-vectorizer-worker-with-docker-compose))
 
-## Prerequisites
 
-To run vectorizer workers, you need to:
+## Running a vectorizer worker as a CLI tool
 
-* Install:
-  * Container environment: [Docker][docker]
-  * Local environment: [Python3][python3] and [pip][pip]
-  * All environments: A Postgres client like [psql][psql]
-* Create a key for your AI provider:
-  * [OpenAI][openai-key]
-  * [Voyage AI][voyage-key]
+**Prerequisites**: [Python3][python3] and [pip][pip]
 
-## Install and configure vectorizer worker
 
-To be able to run vectorizers in your self-hosted database, use one of the following setups:
+1. **Install [pgai](https://pypi.org/project/pgai/) from PyPI**
+
+   ```shell
+   pip install pgai
+   ```
+
+   The `pgai` command line tool should now be in your `$PATH`.
+   
+1. **Create a .env file**
+
+   [Configure](#setting-api-keys-through-environment-variables-or-env-file) the API keys for your embedding providers by adding them to a `.env` file. For example, if you are using OpenAI, you can add the following:
+
+   ```
+   OPENAI_API_KEY=<your-openai-api-key>
+   ```
+   
+   Alternatively, you can set the API key through an environment variable, but this is less secure.
```

Comment:
```suggestion
   Alternatively, you can set the API key through an environment variable directly.
```

I don't think this impact security in a meaningful way. Both are basically clear text passkeys. Yes env vars are available to any program running in that shell but any app that runs also has access to your file system so... idk I'd just leave it out.